Project.toml

```diff
@@ -1,18 +1,20 @@
 name = "PlotUtils"
 uuid = "995b91a9-d308-5afd-9ec6-746e21dbc043"
-version = "0.6.5"
+version = "0.7.0"
```

Comment:
why not 1.0?

Reply:
Hm, no particular reason. 1.0 would also be an option.